lib/pages/chat_list/chat_list.dart

```diff
@@ -79,6 +78,8 @@ class ChatListController extends State<ChatList>
 
   String? activeSpaceId;
 
+  final chatListIndexes = <String, int>{};
```

Comment:
what is it

Reply:
It's a method for optimize the chat list when chat list is reordered